test/controls/tab-bar.test.js

```diff
@@ -9,107 +10,94 @@ const objectOptions = [
-  expect(wrapper.find('li').last().text()).toEqual('Account')
+  render(<TabBar options={defaultOptions} value="home" />)
+  expect(screen.getByText('Home')).toBeInTheDocument()
+  expect(screen.getByText('Account')).toBeInTheDocument()
 })
 
 test('TabBar renders objectOptions', () => {
-  const wrapper = mount(<TabBar options={objectOptions} value="home" />)
-  expect(wrapper.find('li').first().text()).toEqual('Home')
-  expect(wrapper.find('li').last().text()).toEqual('Account')
+  render(<TabBar options={objectOptions} value="home" />)
+  expect(screen.getByText('Home')).toBeInTheDocument()
+  expect(screen.getByText('Account')).toBeInTheDocument()
 })
 
 test('TabBar adds Active class', () => {
-  const wrapper = mount(<TabBar options={objectOptions} value="home" />)
-  expect(wrapper.find('li').first().hasClass('active')).toEqual(true)
+  render(<TabBar options={objectOptions} value="home" />)
+  expect(screen.getByText('Home').parentElement).toHaveClass('active')
 })
 
-test('TabBar calls onChange', () => {
+test('TabBar calls onChange', async () => {
   const onChange = jest.fn()
-  const wrapper = mount(
+  const user = userEvent.setup()
+  render(
     <TabBar options={objectOptions} value="home" onChange={onChange} />
   )
-  wrapper.find('li > a').first().simulate('click')
-  expect(onChange).toHaveBeenCalledWith('home')
+  await user.click(screen.getByText(objectOptions[0].key))
+  expect(onChange).toHaveBeenCalledWith(objectOptions[0].value)
 })
 
 test('TabBar passes down custom className to ul', () => {
-  const wrapper = mount(
+  render(
     <TabBar options={objectOptions} value="home" className="custom" />
   )
-  expect(wrapper.find('ul').hasClass('custom')).toEqual(true)
+  expect(screen.getByRole('tablist')).toHaveClass('tabs', 'custom')
 })
 
 test('TabBar passes down custom activeClassName to li', () => {
-  const wrapper = mount(
+  render(
     <TabBar options={objectOptions} value="home" activeClassName="custom" />
   )
-  expect(wrapper.find('li').first().hasClass('custom')).toEqual(true)
+  const homeTab = screen.getByText('Home')
+  expect(homeTab.parentElement).toHaveClass('custom')
+  expect(homeTab.parentElement).not.toHaveClass('active')
 })
 
 test('TabBar assigns appropriate aria roles', () => {
-  const wrapper = mount(<TabBar options={defaultOptions} value="home" />)
-  expect(wrapper.find('ul').prop('role')).toBe('tablist')
-  expect(wrapper.find('li > a').every('[role="tab"]')).toBe(true)
-})
-
-test('TabBar assigns appropriate aria orientation', () => {
-  const horizontalWrapper = mount(
-    <TabBar options={defaultOptions} vertical={false} value="home" />
-  )
-  const verticalWrapper = mount(
-    <TabBar options={defaultOptions} vertical value="home" />
-  )
-
-  expect(
-    horizontalWrapper.find('[role="tablist"]').prop('aria-orientation')
-  ).toBe('horizontal')
-  expect(
-    verticalWrapper.find('[role="tablist"]').prop('aria-orientation')
-  ).toBe('vertical')
```

Comment:
We don't need this test anymore because we're asserting on these properties in the first two tests (instead of relying on classes)